products/sle/main.pm

```diff
@@ -777,6 +777,7 @@ elsif (get_var("VIRT_AUTOTEST")) {
                 loadtest "virt_autotest/libvirt_isolated_virtual_network";
             }
             loadtest "virt_autotest/setup_dns_service";
+            loadtest "virt_autotest/sriov_pci_passthrough" if get_var("ENABLE_SRIOV_PCI_PASSSHTROUGH");
```

Comment:
Again if this is only to do with network card sriov, please use other more related parameter name, for example, enable_network_sriov_pci_passthrough

Reply:
Yes, together with test name.